tests/test_utils.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA-Server tests for utils module."""
+
+import pytest
+from reana_server.utils import is_valid_email
+
+
+@pytest.mark.parametrize(
+    "email,is_valid",
+    [
+        ("john.doe@example.org", True),
```

Comment:
You may also put the simplest example `john@example.org` to make it explicit that it passes.